api/advanced-api.yaml

```diff
@@ -0,0 +1,114 @@
+paths:
+  /advanced/config:
+    post:
+      tags: ['Advanced']
```

Comment:
Should the tag be 'Advanced Configuration'?

Reply:
Keeping all advanced APIs under the tag `Advanced` rather than two separate tags for config and metrics.  This can easily be changed in the future without incurring any compatibility issues.